### PR TITLE
Adds techdocsmith and vtlim as code owners for documentation related …

### PR DESCRIPTION
**Pull request by techdocsmith**
This PR uses the CODEOWNERS file as a workaround for notifications since GitHub doesn't allow watching a directory within a repo. @techdocsmith and @vtlim are Druid committers focused exclusively on the documentation, so it makes sense for them to be notified on changes. It'll also help catch PRs that need the `Area - Documentation` label applied and doc reviews.

This does not change the repo settings to require a review from code owners, so there's no change to any review requirements for PRs. Any committer will still be able to approve a PR.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Adds techdocsmith and vtlim as code owners for documentation related directories. 
+/docs/** @techdocsmith @vtlim
+/examples/quickstart/jupyter-notebooks/** @techdocsmith @vtlim
```
Comment (paul-rogers) [inline .github/CODEOWNERS:3]: Nit: missing newline

**Review by paul-rogers — APPROVED** [review]
Please fix the missing newline, else LGTM.

File: .github/CODEOWNERS (hunk 1 above)
Comment (techdocsmith) [inline .github/CODEOWNERS:3]:
```suggestion
/examples/quickstart/jupyter-notebooks/** @techdocsmith @vtlim

```

**Comment by techdocsmith** [comment]
Converted to draft while we look at another solution.

**Comment by techdocsmith** [comment]
Superseded by #13791

**Pull request closed without merging**